frontend/src/pages/ProjectDetails.tsx

```diff
@@ -81,23 +81,23 @@ const ProjectDetailsPage = () => {
   const projectStats = [
     {
       icon: faUsers,
-      value: `${typeof project?.contributorsCount === 'number' && project?.contributorsCount > 0 ? millify(project?.contributorsCount, { precision: 1 }) : 'No'} ${pluralize(project.contributorsCount, 'Contributor')}`,
+      value: `${project?.contributorsCount ? millify(project?.contributorsCount, {precision: 1}) : 'No'} ${pluralize(project.contributorsCount, 'Contributor')}`,
```

Comment:
We should simplify this (here and below) as the numbers are always provided.
```suggestion
      value: `${millify(project.contributorsCount, {precision: 1}) : 'No'} ${pluralize(project.contributorsCount, 'Contributor')}`,
```